### PR TITLE
Databases/week2

### PR DESCRIPTION
**Pull request by Hani-far**

### CODE REVIEW

File: databases/week2/Working with tasks.sql [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+WHERE id=37;
+
+-- Mark a task as complete
+UPDATE task
+SET status_id = 3
+WHERE id=37;
+
+-- Delete a task
+DELETE from task
+WHERE id=37;
```
Comment (Elia-Git) [inline databases/week2/Working with tasks.sql:29]: Good Job!

File: databases/week2/School database.sql [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+phone varchar(50),
+class_id int unsigned,
+foreign key (class_id) references Class(id)
+);
+
+CREATE INDEX name
+ON Student (name);
+
+ALTER TABLE Class
+ADD status enum('not-started','ongoing','finished');
```
Comment (Elia-Git) [inline databases/week2/School database.sql:34]: Good Hanieh!

File: databases/week2/Favourite books/Favourite books.sql [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+author_id int unsigned not null,
+book_id int unsigned not null,
+PRIMARY KEY(`author_id`, `book_id`),
+  CONSTRAINT `fk_author_books_author` FOREIGN KEY (`author_id`) REFERENCES `authors` (`id`) ON DELETE CASCADE ON UPDATE CASCADE,
+  CONSTRAINT `fk_author_books_book` FOREIGN KEY (`book_id`) REFERENCES `books` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
+)ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+
+
+
```
Comment (Elia-Git) [inline databases/week2/Favourite books/Favourite books.sql]: Great and big practice, nice ED (Entity diagram), Keep moving! 
Remove line # 4 , You don't need to include it here and remember to remove empty lines( for instance line # 51 to 53).

File: databases/week2/More queries.sql [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+group by Month
+order by Month;
+
+-- or
+
+SELECT EXTRACT(month FROM task.created) AS Month, count(task.id) AS Created_Tasks
+FROM task
+group by Month
+order by Month;
+
```
Comment (Elia-Git) [inline databases/week2/More queries.sql:40]: Good job here as well!

**Review by Elia-Git — APPROVED** [review]
Hi Hanieh,

Thank you for the great work. Few and minnor comments. Read and apply them and then you can merge it.

File: databases/week2/Favourite books/Favourite books.sql (hunk 3 above)
Comment (Hani-far) [reply databases/week2/Favourite books/Favourite books.sql]: Thank you so much for your time, support and feedback. will do that

File: databases/week2/Working with tasks.sql (hunk 1 above)
Comment (Hani-far) [reply databases/week2/Working with tasks.sql:29]: Thanks

File: databases/week2/More queries.sql (hunk 4 above)
Comment (Hani-far) [reply databases/week2/More queries.sql:40]: Thank you

**Pull request merged**